src/server/address-format-reviver.ts

```diff
@@ -0,0 +1,32 @@
+/*
+A reviver is a function that prescribes how the value originally produced by JSON.parse is transformed, before being returned.
+*/
+
+import { formatEthereumAddress } from "speedomatic";
+
+export interface StringToBoolMap {
+  [key: string]: boolean;
+}
+
+export const inputsExpectedAsAddress: StringToBoolMap = {
+  account: true,
+  category: true,
+  creator: true,
+  designatedReporter: true,
+  marketID: true,
+  marketIDs: true,
```

Comment:
I was going to comment that this won't work, but after reading formatEthereumAddress and seeing it works with arrays, looks good.

Reply:
ya. got a unit test for that case also